Svc/ActiveLogger/docs/sdd.md

```diff
@@ -13,8 +13,8 @@ The requirements for `Svc::ActiveLogger` are as follows:
 
 Requirement | Description | Verification Method
 ----------- | ----------- | -------------------
-AL-001 | The `Svc::ActiveLogger` component shall receive events and compose ter events based on event severity. | Unithem into downlink packets. | Inspection; Unit Test
-AL-002 | The `Svc::ActiveLogger` component shall have commands to filt Test
+AL-001 | The `Svc::ActiveLogger` component shall receive events and compose them into downlink packets. | Inspection; Unit Test
+AL-002 | The `Svc::ActiveLogger` component shall have commands to filter events based on event severity. | Unit Test
```

Comment:
I called this out earlier...

It appears that in https://github.com/nasa/fprime/pull/277/commits/c2a621e030290ef6e41d8b61fab685ea5bff26a7 the editor's cursor shifted by one row resulting in text being placed in the wrong spot.